tests/framework/PostProcessors/InterfacedPostProcessor/test_historySetDelay.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/PostProcessors/InterfacedPostProcessor/HistorySetDelay</name>
+    <author>cogljj</author>
+    <created>2021-04-19</created>
+    <classesTested>InterfacedPostProcessor</classesTested>
+    <description>
+      Tests of the HistorySetDelay interfaced post-processor
```

Comment:
Can you improve the description answering to the following questions:
what do we test? What the expected outcomes? input and outputs? the model used?

Reply:
Fixed.